homeassistant/components/webostv/manifest.json

```diff
@@ -1,12 +1,12 @@
 {
   "domain": "webostv",
-  "name": "LG webOS Smart TV",
+  "name": "LG webOS TV",
```

Comment:
This is already updated in the docs and was missing from core: https://github.com/home-assistant/home-assistant.io/blob/871986071edfcb46923059ad6f134dc9a5994109/source/_integrations/webostv.markdown?plain=1#L2